test/api_integration/fixtures/es_archiver/saved_objects/delete_unknown_types/mappings.json

```diff
@@ -523,7 +523,12 @@
         "number_of_shards": "1",
         "priority": "10",
         "refresh_interval": "1s",
-        "routing_partition_size": "1"
+        "routing_partition_size": "1",
+        "mapping": {
+          "total_fields": {
+            "limit": 10000
```

Comment:
The saved objects index uses `"limit": 1500` so please use that instead. If a test somehow exceeds our default limit we would want to know about it because that might indicate a bigger problem.

Reply:
done